docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/banner/banner.twig

```diff
@@ -30,7 +30,7 @@
 {% if content_top1 is not empty or content_top2 is not empty or content_top3 is not empty or section is not empty or title is not empty or content_middle is not empty or content is not empty or content_below is not empty or content_bottom is not empty %}
   <div class="civictheme-banner {{ modifier_class }}">
     <div class="civictheme-banner__wrapper">
-      <div class="civictheme-banner__wrapper__inner" {% if background_image %}style="background-image: url('{{ background_image }} ')"{% endif %}>
+      <div class="civictheme-banner__wrapper__inner {{ blend_mode }}" {% if background_image %}style="background-image: url('{{ background_image }} ')"{% endif %}>
```

Comment:
@joshua-salsadigital 
sorry - I mislead you when I said to write a generator for background classes (I forgot that our Background is not a full-fledged utility component that you can just apply) - the generator will not work.

Could you please do this:
expose `background_image_blend_mode` asa variable  and then 

```
<div class="civictheme-banner__wrapper__inner {{ blend_mode }}" {% if background_image %}style="background-image: url('{{ background_image }} '){% endif %} {% if background_image_blend_mode %}background-image-blend-mode: {{ background_image_blend_mode }}"{% endif %}">
```

And put the mode value as-is taken from the backend field.